### PR TITLE
rename branch master to main

### PR DESCRIPTION
**Pull request by Eomm**
Renamed the `master` branch to `main` following the [GitHub guide](https://github.blog/changelog/2021-01-19-support-for-renaming-an-existing-branch/)

To update your local repository run:

```
git branch -m master main
git fetch origin
git branch -u origin/main main
```

Ref https://github.com/nodejs/package-maintenance/issues/445

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ of support they may intend to provide.  The [Node.js Package Maintenance Working
 is working to propose [a spec](https://github.com/nodejs/package-maintenance/blob/HEAD/docs/PACKAGE-SUPPORT.md) to help package authors declare their intended support goals.  This package provides
 some tooling around working with the format proposed.
 
-This repository is managed by the [Package Maintenance Working Group](https://github.com/nodejs/package-maintenance), see [Governance](https://github.com/nodejs/package-maintenance/blob/master/Governance.md).
+This repository is managed by the [Package Maintenance Working Group](https://github.com/nodejs/package-maintenance), see [Governance](https://github.com/nodejs/package-maintenance/blob/main/Governance.md).
```
Comment (ljharb) [inline README.md]:
```suggestion
This repository is managed by the [Package Maintenance Working Group](https://github.com/nodejs/package-maintenance), see [Governance](https://github.com/nodejs/package-maintenance/blob/HEAD/Governance.md).
```

File: doc/command-line-usage.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ you can run the support tool using `npx @pkgjs/support`.
 
 The support tool helps package consumers review and understand the
 package support information provided by maintainers. See
-[Package-support.md](https://github.com/nodejs/package-maintenance/blob/master/docs/PACKAGE-SUPPORT.md)
+[Package-support.md](https://github.com/nodejs/package-maintenance/blob/main/docs/PACKAGE-SUPPORT.md)
```
Comment (ljharb) [inline doc/command-line-usage.md]:
```suggestion
[Package-support.md](https://github.com/nodejs/package-maintenance/blob/HEAD/docs/PACKAGE-SUPPORT.md)
```

File: doc/command-line-usage.md [hunk 3]
```diff
@@ -97,7 +97,7 @@ npx @pkgjs/support show --canonical
 
 
 ```
-@pkgjs/support(0.0.2) - https://github.com/pkgjs/support/blob/master/package-support.json
+@pkgjs/support(0.0.2) - https://github.com/pkgjs/support/blob/main/package-support.json
```
Comment (ljharb) [inline doc/command-line-usage.md]:
```suggestion
@pkgjs/support(0.0.2) - https://github.com/pkgjs/support/blob/HEAD/package-support.json
```

**Review by ljharb — COMMENTED** [review]
s/main/HEAD throughout on the GitHub urls

**Review by rodion-arr — APPROVED** [review]

File: doc/command-line-usage.md [hunk 4]
```diff
@@ -116,7 +116,7 @@ npx @pkgjs/support show --canonical
 ```
 
 where the support info is listed as 
-`https://github.com/pkgjs/support/blob/master/package-support.json`
+`https://github.com/pkgjs/support/blob/main/package-support.json`
```
Comment (ljharb) [inline doc/command-line-usage.md]: there’s a few more in this doc
```suggestion
`https://github.com/pkgjs/support/blob/HEAD/package-support.json`
```

**Review by mhdawson — APPROVED** [review]
LGTM

File: doc/command-line-usage.md (hunk 4 above)
Comment (Eomm) [reply doc/command-line-usage.md]: Done!

File: doc/command-line-usage.md [hunk 5]
```diff
@@ -177,15 +177,15 @@ The `validate` command checks that both the information added to the package.jso
 and the file containing the support info (package-support.json by default) is valid.
 
 
-[Integration into package.json](https://github.com/nodejs/package-maintenance/blob/master/docs/PACKAGE-SUPPORT.md#integration-into-packagejson)
+[Integration into package.json](https://github.com/nodejs/package-maintenance/blob/main/docs/PACKAGE-SUPPORT.md#integration-into-packagejson)
```
Comment (ljharb) [inline doc/command-line-usage.md]:
```suggestion
[Integration into package.json](https://github.com/nodejs/package-maintenance/blob/HEAD/docs/PACKAGE-SUPPORT.md#integration-into-packagejson)
```

File: doc/command-line-usage.md [hunk 6]
```diff
@@ -177,15 +177,15 @@ The `validate` command checks that both the information added to the package.jso
...
 explains the options for providing support info and what goes into the package.json for
 each case.
 
 `npx @pkgjs/support validate` is run from the directory that contains the package.json for the package.
 Depending the how the `support` section in the package.json is configured validate will:
 
 * validate the format of the `support` section in the package.json conforms to the
-documentation in [Integration into package.json](https://github.com/nodejs/package-maintenance/blob/master/docs/PACKAGE-SUPPORT.md#integration-into-packagejson)
+documentation in [Integration into package.json](https://github.com/nodejs/package-maintenance/blob/main/docs/PACKAGE-SUPPORT.md#integration-into-packagejson)
```
Comment (ljharb) [inline doc/command-line-usage.md]:
```suggestion
documentation in [Integration into package.json](https://github.com/nodejs/package-maintenance/blob/HEAD/docs/PACKAGE-SUPPORT.md#integration-into-packagejson)
```

**Pull request merged**